docs/soar_manual/04_ProceduralKnowledgeLearning.md

```diff
@@ -1371,9 +1375,9 @@ call any working memory element that is guaranteed to only have a single
 possible value at any given time, a singleton. If EBC encounters two different
 conditions in the backtrace that both test the same superstate WME that matches
 a user singleton pattern, it will merge the two conditions. As described in
-Section 4b, there are several architectural singleton’s that EBC already knows
-about. To specify patterns for domain-specific singletons, the chunk
-singleton command can be used.
+[Listing 4b](#identity-assignment-and-propagation), there are several
```

Comment:
On the website it appears to be a section, not a listing (which would be a single code example). Also the section is not currently numbered, so 4b doesn't mean anything. I did open a ticket for this: https://github.com/SoarGroup/SoarGroup.github.io/issues/38

Reply:
I am aware of that, but in the latex/ pdf version the link references the listing and this was the closest to replicate that link in Markdown since there is no way to reference listings in Markdown AFAIK.

I am open to other suggestions, though!

Reply:
What about the side-by-side listing you added in chapter 4 for showing the 9.4.0 chunk vs. the new EBC chunk? Perhaps you could do a similar thing for this listing, then give it an ID to make it linkable?

Reply:
I ended up using HTML ids and link in Markdown for this (although it is removed with the usage section anyway, see #41) and the image links.